content/docs/how-to-contribute.md

```diff
@@ -9,15 +9,15 @@ redirect_from:
   - "tips/introduction.html"
 ---
 
-React is one of Facebook's first open source projects that is both under very active development and is also being used to ship code to everybody on [facebook.com](https://www.facebook.com). We're still working out the kinks to make contributing to this project as easy and transparent as possible, but we're not quite there yet. Hopefully this document makes the process for contributing clear and answers some questions that you may have.
+تعتبر React من أوائل المشاريع مفتوحة المصدر والتي ﻻزالت تحت التطوير المستمر والتي يتم استعمالها في شفرات برمجية للجميع في [facebook.com](https://www.facebook.com). ﻻزلنا مستمرّين في العمل على إصلاح العلل لجعل المُساهمة في هذا المشروع أسهل وأكثر شفافية قدر الإمكان. لم نبلغ ذلك حقّا، لكن نتمنى أن يوضّح هذا المستند مراحل المساهمة ويجيب عن الأسئلة التي يمكن أن تُراودك.
```

Comment:
I think the repeat of `والتي` is bad. 
So, i suggest to replace the second `والتي` with `حيث`.

Reply:
It will have a different meaning, some projects are in active development but they're not being used in production... So `حيث` will give a different meaning.

Reply:
Read the context in Arabic , you will find that it has the same meaning ..
The idea here is to avoid repetition. If you have another word or a better context, that is good

Reply:
@Fcmam5 !!